alvr/dashboard/js/app/monitor.js

```diff
@@ -633,70 +626,14 @@ define([
                     spanGaps: false,
                 },
                 {
-                    label: "Pre Submit",
-                    stroke: graphColors[1],
-                    fill: graphColors[1],
-                    value: (u, v, si, i) => (latencyGraphData[si][i] || 0).toFixed(3) + " ms",
-                    spanGaps: false,
-                },
-                {
-                    label: "Post Submit",
-                    stroke: graphColors[1],
-                    fill: graphColors[1],
-                    value: (u, v, si, i) => (latencyGraphData[si][i] || 0).toFixed(3) + " ms",
-                    spanGaps: false,
-                },
-                {
-                    label: "Total Render",
-                    stroke: graphColors[1],
-                    fill: graphColors[1],
-                    value: (u, v, si, i) => (latencyGraphData[si][i] || 0).toFixed(3) + " ms",
-                    spanGaps: false,
-                },
-                {
-                    label: "Compositor (GPU)",
+                    label: "Render",
                     stroke: graphColors[1],
                     fill: graphColors[1],
                     value: (u, v, si, i) => (latencyGraphData[si][i] || 0).toFixed(3) + " ms",
                     spanGaps: false,
                 },
```

Comment:
Lots of duplication; can you either:
- make a function with a signature like this? `(graphColor, label) => { label, stroke, fill, value, spanGaps }`
- use [`Array#map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to populate the fields that stay the same in everything but the total latency?